src/context/videoContext.js

```diff
@@ -1,6 +1,6 @@
 import { createContext, useContext, useReducer } from "react";
 import { useEffect } from "react";
```

Comment:
useEffect to move upwards with other hooks

Reply:
Okay, will do that.